ibm_mq/datadog_checks/ibm_mq/utils.py

```diff
@@ -13,3 +18,25 @@ def sanitize_strings(s):
     if found >= 0:
         s = s[:found]
     return s.strip()
+
+
+def calculate_elapsed_time(datestamp, timestamp, current_time=None):
```

Comment:
The `current_time` parameter was added for testing purposes so that a hard-coded "current" timestamp can be passed in to calculate elapsed time. This is to avoid any potential OverflowErrors returned from `datetime.timestamp()`. Would it be better to remove this parameter and instead test with randomly generated timestamps and the actual current timestamp?

Reply:
I think is good